extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -120,28 +124,39 @@ class AmpStickyAd extends AMP.BaseElement {
   }
 
   /**
-   * The listener function that listen on onScroll event and
-   * show sticky ad when user scroll at least one viewport and
-   * there is at least one more viewport available.
+   * The listener function that listen on viewport scroll event.
+   * And decide when to display the ad.
    * @private
    */
-  displayAfterScroll_() {
+  onScroll_() {
+    if (isExperimentOn(this.win, TAG)) {
+      this.displayAfterScroll_();
```

Comment:
As agreed https://github.com/ampproject/amphtml/issues/6597#issuecomment-278798116.We want to show the ad after first user interaction. As long as scroll slows down and resources manager can handle this mutate, it is OK to display before scroll fully stops.
`#deferMutate` already does the work for me.